command/server.go

```diff
@@ -1760,6 +1760,24 @@ func (c *ServerCommand) Run(args []string) int {
 		case <-c.SigUSR2Ch:
 			logWriter := c.logger.StandardWriter(&hclog.StandardLoggerOptions{})
 			pprof.Lookup("goroutine").WriteTo(logWriter, 2)
+
+			if os.Getenv("VAULT_WRITE_STACKTRACE_TO_FILE") != "" {
+				c.logger.Info("Writing stacktrace to file..")
+
+				tmpFile, err := ioutil.TempFile("", fmt.Sprintf("%s-", filepath.Base(os.Args[0])))
```

Comment:
Seems like ioutil.TemFile has been deprecated. Shall we use `os.CreateTemp` instead?

Reply:
Good catch, fixed!